crates/core/src/client/constants.rs

```diff
@@ -39,3 +39,22 @@ pub mod gas {
     /// As a result, the priority fee is set to 0.
     pub const MAX_PRIORITY_FEE_PER_GAS: U128 = U128::ZERO;
 }
+
+// This module contains constants which are being used in place of real data that should be fetched
+// in production.
+pub mod mock_evm_data {
+    use reth_primitives::{H256, H64};
+
+    pub const GAS_LIMIT: u64 = 1_000_000;
+    pub const GAS_USED: u64 = 500_000;
+    pub const CUMULATIVE_GAS_USED: u64 = 1_000_000;
+    pub const EFFECTIVE_GAS_PRICE: u64 = 1_000_000;
+    pub const SIZE: u64 = 1_000_000;
+    pub const MAX_FEE: u64 = 100_000_000_000_000_000;
+    pub const ESTIMATE_GAS: u64 = 100_000_000_000_000_000;
+    pub const TRANSACTION_TYPE: u64 = 0;
+    pub const NONCE: H64 = H64::zero();
```

Comment:
thinking out loud, maybe this should be None as well, always.

Reply:
Are you talking about `TRANSACTION_TYPE`?

If that is the case, then can you expand on why it should be None?

Reply:
Oh! Gotcha, you are talking about `NONCE`, yes, makes sense!

Reply:
yes Nonce,
I guess it can be in the same issue as Total_Difficulty then

Reply:
Gotcha!